rules/REQUEST-941-APPLICATION-ATTACK-XSS.conf

```diff
@@ -23,7 +23,6 @@ SecRule TX:PARANOIA_LEVEL "@lt 1" "phase:2,id:941012,nolog,pass,skipAfter:END-RE
 #
 # -=[ Libinjection - XSS Detection ]=-
 #
-# Ref: https://libinjection.client9.com/
```

Comment:
This should probably link to their github
https://github.com/client9/libinjection